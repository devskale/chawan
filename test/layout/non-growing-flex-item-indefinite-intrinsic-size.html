<!DOCTYPE html>
<div style="display: flex">
<div>flex item 1</div>
<div style="position: relative; background: pink">
<div style="display: inline-block; width: 100%; background: red">a</div>
<!--
this part is a more or less unrelated test for whether setting relative
on an inline root block box marks it as the containing block.
-->
<div style="right: 0; position: absolute">
absolute
