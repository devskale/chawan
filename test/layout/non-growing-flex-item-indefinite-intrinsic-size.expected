flex item 1[48;2;255;0;0ma[49m
    absolute
